src/views/components/EvaluationForm/index.tsx

```diff
@@ -250,16 +264,12 @@ export default class EvaluationForm extends React.Component<Props, State> {
         </Form.Group>
 
         <Form.Group widths="equal">
-          <Form.Input
-            type="text"
+          <SearchableDropdown
```

Comment:
Wrap it in an inline FormField such that it is not a fat input element anymore.

```suggestion
           <Form.Field inline={true}> <SearchableDropdown ....
```